src/Model/Behavior/SearchBehavior.php

```diff
@@ -120,7 +120,7 @@ public function searchManager()
     protected function _extractParams($params, $filters)
     {
         return array_intersect_key(Hash::filter($params, function ($val) {
-            return !in_array($val, $this->_defaultConfig['emptyValues'], true);
+            return !in_array($val, $this->getConfig('emptyValues'), true);
```

Comment:
Can we trust this to be always an array?
Otherwise I would store in a local `$emptyValues = (array)...` with a cast first.

Reply:
Good point